android/src/main/java/co/nimblehq/ic/kmm/suv/android/ui/screens/surveyquestions/SurveyQuestionsScreen.kt

```diff
@@ -367,15 +450,17 @@ fun SurveyQuestionsScreenPreview(
                 QuestionContentUiModel(
                     "1/2",
                     "What your name?",
-                    displayType = QuestionDisplayType.Intro
+                    displayType = QuestionDisplayType.Intro()
                 ),
                 QuestionContentUiModel(
                     "2/2",
                     "How old are you?",
-                    displayType = QuestionDisplayType.Intro
+                    displayType = QuestionDisplayType.Intro()
                 )
             )
         ),
-        onAnswerChange = {}
+        onAnswerChange = {},
+        onAnswersSubmit = {},
+        isSubmitting = true
```

Comment:
Submitting is different from Loading on this screen.
Loading: load data from the API: I show the shimmer indicator
Submitting: submit the answers: I show the loading on the button only

On the other hand, Compose Preview only support one `@PreviewParameter`, so I just put true for `isSubmitting` as I want to test that view with preview.

Reply:
@suho got the point.

> On the other hand, Compose Preview only support one @PreviewParameter, so I just put true for isSubmitting as I want to test that view with preview.

FYI In case you want to preview more things, you can create a data class instead of a single Boolean for multiple preview fields.